tests/TestCase/Model/Behavior/SearchBehaviorTest.php

```diff
@@ -36,6 +36,21 @@ public function searchConfiguration()
     }
 }
 
+class GroupsTable extends Table
```

Comment:
Ideally, all those classes should be moved into the tests/TestApp/ directory as standalone class files.
But we can do that in a future cleanup PR as well.